draft-ietf-quic-qpack.md

```diff
@@ -587,8 +585,10 @@ d = count of entries dropped
 ### Prefixed Integers
 
 The prefixed integer from Section 5.1 of [RFC7541] is used heavily throughout
-this document.  The format from [RFC7541] is used unmodified.  QPACK
-implementations MUST be able to decode integers up to 62 bits long.
+this document.  The format from [RFC7541] is used unmodified.  Note, however,
+that QPACK uses some prefix sizes not actually used in HPACK.
+
+QPACK implementations MUST be able to decode integers up to 62 bits long.
```

Comment:
As long as we are being pedantic: "up to *and including*?"

Reply:
Created issue #3134